### PR TITLE
[NTOS:KDBG] Fixed variable declaration to make buildable the project

### PR DESCRIPTION
**Pull request by adamantinum**
Moved a declaration inside the #ifndef

### CODE REVIEW

**Comment by SergeGautherie** [comment]
Commit is wrong: end-of-line mismatch?

**Comment by adamantinum** [comment]
@binarymaster sorry, this is my first PR and I'm not able to use git very well... Following your link, I didn't understand what is <earliercommit> in git rebase command. I suppose it is the string in "partents", but it doesn't work.

**Comment by binarymaster** [comment]
> I didn't understand what is in git rebase command.

You can ask for help in the chat. Also please fix line endings.

File: ntoskrnl/kdbg/kdb.c [hunk 1]
```diff
@@ -1605,11 +1605,38 @@ KdbEnterDebuggerException(
             ExceptionRecord && ExceptionRecord->NumberParameters != 0)
         {
             /* FIXME: Add noexec memory stuff */
+<<<<<<< HEAD
```
Comment (SergeGautherie) [inline ntoskrnl/kdbg/kdb.c:1608]: Hum!

Comment (adamantinum) [reply ntoskrnl/kdbg/kdb.c]: Yes, I noticed it too late, I'm fixing...

File: ntoskrnl/kdbg/kdb.c [hunk 2]
```diff
@@ -1606,8 +1606,9 @@ KdbEnterDebuggerException(
         {
             /* FIXME: Add noexec memory stuff */
             ULONG_PTR TrapCr2;
+#ifndef _M_AMD64
             ULONG Err;
-
```
Comment (SergeGautherie) [inline ntoskrnl/kdbg/kdb.c:1610]: Please, do not remove the blank lline.

**Comment by adamantinum** [comment]
Hi, now I think should be all fixed.

**Comment by SergeGautherie** [comment]
Commit message format is still wrong.

**Comment by adamantinum** [comment]
Is it quite good?

**Comment by adamantinum** [comment]
Added also module to the commit

**Comment by Extravert-ir** [comment]
Congratulations with the first PR!

**Comment by adamantinum** [comment]
Thanks!

**Pull request merged**